packages/client/src/components/GrantsTableNext.vue

```diff
@@ -16,10 +16,7 @@
         <h4 class="mb-0">{{ searchTitle }}</h4>
       </b-col>
       <b-col align-self="end">
-        <a href="#" @click="exportCSV" :disabled="loading" variant="outline-primary border-0"
-          class="text-right text-nowrap">
-          <p class="mb-0">Export CSV</p>
-        </a>
+        <p class="mb-0 text-right text-nowrap"><a href="#" @click="exportCSV" :disabled="loading" variant="outline-primary border-0">Export CSV</a></p>
```

Comment:
small css change that ended up revealing itself when testing award-ceiling since the entire <p> tag was a link, when I was clicking award-ceiling it would end up downloading a CSV if I mis-clicked by a hair.